packages/framework/src/types/schema.types.ts

```diff
@@ -1,7 +1,7 @@
 import type { JSONSchema, FromSchema as JsonSchemaInfer } from 'json-schema-to-ts';
 import zod from 'zod';
 
-export type JsonSchema = JSONSchema;
+export type JsonSchema = Exclude<JSONSchema, boolean>;
```

Comment:
The JSON schema used in Framework is a root-level JSON schema, which can't be a `boolean`. For completeness, the `boolean` value in the library package is present to allow for `additionalProperties: JSONSchema | boolean`, which is a valid JSON schema polymorphism.